client/src/components/DefaultButton.md

```diff
@@ -0,0 +1,6 @@
+#DefaultButton
+This is a documentation about the Default button.
```

Comment:
As styleguidist is a documentation tool, I think this might not be needed.
Maybe you can add some info about where to use it? What can be provided to the button as props? stuff like that :)